src/org/omegat/Bundle.properties

```diff
@@ -2722,9 +2723,9 @@ ALIGNER_PANEL_LABEL_COUNTER=Counter:
 ALIGNER_PANEL_LABEL_COMPARISON=Comparison Mode:
 ALIGNER_PANEL_LABEL_AVGSCORE=Average Score: {0}
 # BLACK UP-POINTING TRIANGLE
-ALIGNER_PANEL_BUTTON_MOVEUP=\u25b2
+ALIGNER_PANEL_BUTTON_MOVEUP=\u25B2
```

Comment:
What are these capitalization changes in Unicode escapes? Did a tool do this automatically?

Reply:
A tool automatically do that.
I've pushed again without modifications.